<launch>
    <node pkg="pose_estimation" type="yolo_pose_2d_node.py" name="pose_2d_node" output="screen" />
    <node pkg="pose_estimation" type="pose_2d_to_3d_node" name="pose_2d_to_3d_node" output="screen" />
    <node pkg="pose_estimation" type="pointing_detector_node" name="pointing_detector_node" output="screen" />  
</launch>
